src/components/molecules/NavbarSchedule/NavbarSchedule.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback, useState } from "react";
+import classNames from "classnames";
+
+import { currentVenueSelectorData, parentVenueSelector } from "utils/selectors";
+
+import { useSelector } from "hooks/useSelector";
+
+import { SchedulePageModal } from "components/organisms/SchedulePageModal/SchedulePageModal";
+
+import "./NavbarSchedule.scss";
+
+export const NavbarSchedule: React.FC = () => {
+  const venue = useSelector(currentVenueSelectorData);
+  const parentVenue = useSelector(parentVenueSelector);
+
+  const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
+
+  const toggleEventSchedule = useCallback(() => {
+    setEventScheduleVisible((scheduleVisible) => !scheduleVisible);
+  }, []);
+
+  const hideEventSchedule = useCallback(() => {
+    setEventScheduleVisible(false);
+  }, []);
+
+  // TODO: ideally this would find the top most parent of parents and use those details
+  const navbarTitle = parentVenue?.name ?? venue?.name ?? "";
+
+  return (
+    <>
+      <div
+        className={classNames("nav-party-logo", {
+          "nav-party-logo--visible": isEventScheduleVisible,
+        })}
+        onClick={toggleEventSchedule}
+      >
+        {navbarTitle} Schedule
+      </div>
```

Comment:
What is the `nav-party-logo` used for/etc? It sounds like it should be another logo? If so, presumably this bit of code would belong directly in the `NavBar` component?

Alternately, if it's unused, can we remove it?

Either way, aside from the fact that it's currently wrapping the schedule text here, it seems unrelated to this component?

Reply:
The name is missleading. It was copied from the design repo. I'll fix this.